libs/styles/styles.css

```diff
@@ -160,6 +160,10 @@
   --body-font-family: adobe-clean-han-traditional, 'Adobe Clean', adobe-clean, 'Trebuchet MS', sans-serif;
 }
 
+:root:lang(th) {
+  --body-font-family: adobe-clean-thai, 'Adobe Clean', adobe-clean, 'Trebuchet MS', sans-serif;
+}
+
```

Comment:
Posing a thought, would there be any sense in loading a language relevant CSS file? If we'd have a new definition for each language, just the body font families would be a couple hundred lines of CSS.

Currently it's manageable, but wondering nonetheless